app/controllers/templates_controller.rb

```diff
@@ -109,8 +109,8 @@ def resource_name
 
   private
 
-  def safe_render(template)
-    render :plain => @host.render_template(template: template, params: params)
+  def safe_render(template, mode = Foreman::Renderer::REAL_MODE)
+    render :plain => template.render(host: @host, params: params, mode: mode, template: template)
```

Comment:
@ares: Can you please also change this in the unattended controller to be consistent? Thanks.

Reply:
pushed in separate commit, good point